build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.1.0'
+        classpath 'com.android.tools.build:gradle:2.0.0'
```

Comment:
Yeah, the problem is it wasn't even building with the older plugin, therefore Paul and I needed to do this here.